contrib/auto_explain/sql/memory_leaks.sql

```diff
@@ -0,0 +1,41 @@
+CREATE SCHEMA auto_explain_mem_leak_test;
+SET search_path=auto_explain_mem_leak_test;
+
+LOAD 'auto_explain';
+SET auto_explain.log_nested_statements=true;
+SET auto_explain.log_min_duration = 0;
+SET auto_explain.log_analyze = true;
+
+CREATE OR REPLACE FUNCTION get_executor_mem(num int) returns int 
+language plpgsql
+as $$
+declare
+    line text;
+    mem int[];
+    total int default 0;
+begin
+    for line in execute('EXPLAIN ANALYZE 
+        SELECT information_schema._pg_truetypid(a, t)
+        FROM (
+            SELECT a as a, t as t
+            FROM pg_type t
+                JOIN pg_attribute a ON a.atttypid = t.oid
+            WHERE typtype = ''d''
+            LIMIT 1
+        ) at 
+            JOIN generate_series(1, ' || num ||') gc ON true;')
```

Comment:
Minor, but a good way is to use bind variable here.
And probably, this line is misformatted.

Reply:
done